### PR TITLE
feat(tianmu):Improve the readbility of stonedb.(#11)

### PR DESCRIPTION
**Pull request by DandreChen**
## Summary about this PR

Issue Number: close #11 

[summary]
class member variable refactor：handler directory

## Tests Check List

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [x] New Feature
- [ ] Bug Fix
- [ ] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: storage/tianmu/handler/ha_rcengine.cpp [hunk 1]
```diff
@@ -130,19 +130,19 @@ int TIANMU_LoadData(THD *thd, sql_exchange *ex, TABLE_LIST *table_list, void *ar
 
 // returning true means 'to continue'
 bool tianmu_load(THD *thd, sql_exchange *ex, TABLE_LIST *table_list, void *arg) {
-  char tianmumsg[256];
+  char tianmu_msg[256];
```
Comment (RingsC) [inline storage/tianmu/handler/ha_rcengine.cpp]: the variables should be initialized.

**Review by RingsC — CHANGES_REQUESTED** [review]

File: storage/tianmu/handler/ha_rcengine.cpp (hunk 1 above)
Comment (DandreChen) [reply storage/tianmu/handler/ha_rcengine.cpp]: fixed it

File: storage/tianmu/handler/ha_rcengine.cpp [hunk 2]
```diff
@@ -130,19 +130,19 @@ int TIANMU_LoadData(THD *thd, sql_exchange *ex, TABLE_LIST *table_list, void *ar
 
 // returning true means 'to continue'
 bool tianmu_load(THD *thd, sql_exchange *ex, TABLE_LIST *table_list, void *arg) {
-  char tianmumsg[256];
+  char tianmu_msg[256] = {0};
   int tianmu_errcode;
```
Comment (RingsC) [inline storage/tianmu/handler/ha_rcengine.cpp]: How about this `tianmu_errcode` ?

**Review by RingsC — CHANGES_REQUESTED** [review]

File: storage/tianmu/handler/tianmu_handler.cpp [hunk 3]
```diff
@@ -1467,14 +1475,14 @@ int TianmuHandler::reset() {
...
-    table_new_iter_end = core::RCTable::Iterator();
-    table_ptr = NULL;
-    filter_ptr.reset();
-    m_query.reset();
-    m_cq.reset();
-    m_result = false;
-    blob_buffers.resize(0);
+    table_new_iter_ = core::RCTable::Iterator();
+    table_new_iter_end_ = core::RCTable::Iterator();
+    table_ptr_ = NULL;
```
Comment (RingsC) [inline storage/tianmu/handler/tianmu_handler.cpp]: recommend: nullptr.

File: storage/tianmu/handler/tianmu_handler.h [hunk 4]
```diff
@@ -164,24 +164,24 @@ class TianmuHandler final : public handler {
...
 
-  std::shared_ptr<core::TableShare> share;
+  std::shared_ptr<core::TableShare> share_;
 
-  THR_LOCK_DATA m_lock; /* MySQL lock */
-  std::string m_table_name;
-  uint m_dupkey_pos = -1;
+  THR_LOCK_DATA lock_; /* MySQL lock */
+  std::string table_name_;
+  uint dupkey_pos_ = -1;
```
Comment (RingsC) [inline storage/tianmu/handler/tianmu_handler.h]: assign negtive integer to an unsigned int may lead turn-flip. Becare about the symbol flag.

File: storage/tianmu/handler/tianmu_handler.h [hunk 5]
```diff
@@ -164,24 +164,24 @@ class TianmuHandler final : public handler {
...
 
-  core::JustATable *table_ptr = nullptr;
-  std::unique_ptr<core::Filter> filter_ptr;
-  uint64_t current_position;
+  core::JustATable *table_ptr_ = nullptr;
+  std::unique_ptr<core::Filter> filter_ptr_;
```
Comment (RingsC) [inline storage/tianmu/handler/tianmu_handler.h:174]: All these variables should have initial values.

**Review by RingsC — CHANGES_REQUESTED** [review]

File: storage/tianmu/handler/ha_rcengine.cpp (hunk 2 above)
Comment (DandreChen) [reply storage/tianmu/handler/ha_rcengine.cpp]: fixed it

**Review by RingsC — APPROVED** [review]
LGTM

**Review by lujiashun — APPROVED** [review]
LGTM

**Pull request merged**